Examples/Scripts/PostFreeSurferPipelineBatch-long.sh

```diff
@@ -0,0 +1,176 @@
+#Excluded timepoint directories
+ExcludeVisits=(HCA6002237_V1_MR HCA6002238_V1_MR)
+#Longitudinal template labels, one per each subject.
+Templates=(HCA6002236_V1_V2 HCA6002237_V1_V2 HCA6002238_V1_V2)
+#EnvironmentScript="${HOME}/projects/Pipelines/Examples/Scripts/SetUpHCPPipeline.sh" #Pipeline environment script
+EnvironmentScript="${HOME}/projects/Pipelines/Examples/Scripts/SetUpHCPPipeline.sh"
+source "$EnvironmentScript"
+
+##################################################################################################
+# Input variables used by PostFreesurferPipelineLongPrep
+##################################################################################################
+# Hires T1w MNI template
+T1wTemplate="${HCPPIPEDIR_Templates}/MNI152_T1_0.8mm.nii.gz"
+# Hires brain extracted MNI template1
+T1wTemplateBrain="${HCPPIPEDIR_Templates}/MNI152_T1_0.8mm_brain.nii.gz"
+# Lowres T1w MNI template
+T1wTemplate2mm="${HCPPIPEDIR_Templates}/MNI152_T1_2mm.nii.gz"
+# Hires T2w MNI Template
+T2wTemplate="${HCPPIPEDIR_Templates}/MNI152_T2_0.8mm.nii.gz"
+# Hires T2w brain extracted MNI Template
+T2wTemplateBrain="${HCPPIPEDIR_Templates}/MNI152_T2_0.8mm_brain.nii.gz"
+# Lowres T2w MNI Template
+T2wTemplate2mm="${HCPPIPEDIR_Templates}/MNI152_T2_2mm.nii.gz"
+# Hires MNI brain mask template
+TemplateMask="${HCPPIPEDIR_Templates}/MNI152_T1_0.8mm_brain_mask.nii.gz"
+# Lowres MNI brain mask template
+Template2mmMask="${HCPPIPEDIR_Templates}/MNI152_T1_2mm_brain_mask_dil.nii.gz"
+# FNIRT 2mm T1w Config
+FNIRTConfig="${HCPPIPEDIR_Config}/T1_2_MNI152_2mm.cnf"
+
+##################################################################################################
+# Input variables used by PostFreesurferPipeline (longitudinal mode)
+##################################################################################################
+SurfaceAtlasDIR="${HCPPIPEDIR_Templates}/standard_mesh_atlases"
+GrayordinatesSpaceDIR="${HCPPIPEDIR_Templates}/91282_Greyordinates"
+GrayordinatesResolutions="2" #Usually 2mm, if multiple delimit with @, must already exist in templates dir
+HighResMesh="164" #Usually 164k vertices
+LowResMeshes="32" #Usually 32k vertices, if multiple delimit with @, must already exist in templates dir
+SubcorticalGrayLabels="${HCPPIPEDIR_Config}/FreeSurferSubcorticalLabelTableLut.txt"
+FreeSurferLabels="${HCPPIPEDIR_Config}/FreeSurferAllLut.txt"
+ReferenceMyelinMaps="${HCPPIPEDIR_Templates}/standard_mesh_atlases/Conte69.MyelinMap_BC.164k_fs_LR.dscalar.nii"
+RegName="MSMSulc" #MSMSulc is recommended, if binary is not available use FS (FreeSurfer)
+
+# Requirements for this script
+#  installed versions of: FSL, Connectome Workbench (wb_command)
+#  environment: HCPPIPEDIR, FSLDIR, CARET7DIR
+
+#fslsub queue
+#QUEUE=short.q
+#top level parallelization
+
+QUEUE="long.q"
+
+#pipeline level parallelization
+#parallel mode: FSLSUB, BUILTIN, NONE
+parallel_mode=FSLSUB
```

Comment:
I don't think FSLSUB should be the default for other users, in particular qunex can't support it.

Reply:
agree